### ISSUE TITLE
export and display specific columns from MSA

### ISSUE THREAD

**Issue by Yogesh1-11** [GMOD/react-msaview#61]
hi 
one feature that i always wanted in a msaviewer is to diplay  active site residue alignment. can you please implement exporting some sites from MSA for comparative display in figures?

**Comment by cmdcolin** [comment]
just to be sure, you are referring to like, SVG type export for figures? I agree this would be great

**Comment by Yogesh1-11** [comment]
yes. making a new group of  selected columns(from aligned MSA) can help. I am having hard time finding a msa viewer with active site export along with phylogenetic tree on LHS next to header.

**Comment by cmdcolin** [comment]
didn't want this to languish too long so I made a proof of concept of this, can test it out on https://gmod.github.io/react-msaview or download latest from npm, however you are using it

can use "Export SVG" from dropdown

![image](https://github.com/GMOD/react-msaview/assets/6511937/a00f9216-124d-4ef7-b2a8-06fb5e39a0fe)


will probably add things like ability to export the minimap header bar, add coordinates, etc also. let me know if that helps

**Comment by cmdcolin** [comment]
@Yogesh1-11 did the svg export solve the issue? I have a feeling you were requesting something else

**Comment by Yogesh1-11** [comment]
Yes Dr. Diesh
 svg export is good but i was also looking for options to display selected
columns from alignment. for example show only 5th and 10th position from
alignment and hide 6th,7th,8th 9th. I did it with ete3 or i will have to
make a new fasta with selected positions only. but it would be great if
there was a feature to hide undesired positions from lengthy alignments
